Linux-Application/DomesdayDuplicator/usbcapture.cpp

```diff
@@ -29,7 +29,12 @@
 
 #include <atomic>
 #include <sched.h>
-#include <sys/mman.h>
+#ifdef __linux__
```

Comment:
None of this is Linux-specific - it's all standard POSIX features. So this should be `#ifdef _WIN32` new code `#else` existing code `#endif` (assuming Windows doesn't support the POSIX API, which is a bit surprising).

Reply:
Corrected.

Note about Windows - it's not fully POSIX compliant.  Page lock functionality is implemented in the Windows API via `memoryapi.h`.  Task scheduling doesn't have a 1:1 alternative in the API, so that part will either be written to leverage `processthreadsapi.h` (most likely) or use a ported version of the POSIX threads (like `pthread-win32`, but I'm less likely to use this)